modules/auto/index.ts

```diff
@@ -140,10 +142,29 @@ async function handleMutatable(message: Message) {
 			const embed = await handleMatch(match);
 			if (embed) {
 				embeds.push(embed);
-				if (notSet) embed.footer = { text: "Disable this using /settings" };
 			}
```

Comment:
```suggestion
				if (!notSet) embed.footer = { text: "Disable this using /settings" };
			}
```
for consistency with xp